# Encoder status for one channel in a Phidgets High-Speed Encoder board.
Header header
# Tick count (position) for this encoder channel
int32 count
# Instantaneous speed estimation [ticks/s].
# See also: EncoderDecimatedSpeed
float64 inst_vel
